vars/githubscm.groovy

```diff
@@ -100,18 +100,20 @@ def commitChanges(String commitMessage, String filesToAdd = '--all') {
 
 def forkRepo(String credentialID = 'kie-ci') {
     cleanHubAuth()
-    withCredentials([usernamePassword(credentialsId: credentialID, usernameVariable: 'GITHUB_USER', passwordVariable: 'GITHUB_PASSWORD')]) {
-        sh 'git config --global hub.protocol https'
+    withCredentials([usernamePassword(credentialsId: credentialID, usernameVariable: 'GITHUB_USER', passwordVariable: 'GITHUB_TOKEN')]) {
```

Comment:
We should probably also update the credentials in Jenkins to hold the tokens instead of passwords, which are used in the following section of the release pipeline:
https://github.com/kiegroup/kogito-pipelines/blob/master/Jenkinsfile.release#L71-L75

Reply:
`kie-ci` is configured now with a token

Reply:
maybe not the `bsig-gh-bot` ...